src/controls/MMapGeolocationControl/index.ts

```diff
@@ -131,6 +131,8 @@ class MMapGeolocationControl extends mappable.MMapGroupEntity<MMapGeolocationCon
     }
 
     protected override _onDetach() {
+        this._control.removeChild(this._button);
```

Comment:
don't forget to delete all entities created in `_onAttach`

or move its to constructor